cmd/glasskube/cmd/list.go

```diff
@@ -95,6 +97,8 @@ func init() {
 		"show the (cluster-)package description")
 	listCmd.PersistentFlags().BoolVar(&listCmdOptions.ShowLatestVersion, "show-latest", false,
 		"show the latest version of (cluster-)packages if available")
+	listCmd.PersistentFlags().BoolVar(&listCmdOptions.ShowMessage, "show-message", false,
```

Comment:
since it is included in `--more`, they should be mutually exclusive too, like `--show-latest`. So please mark them mutually exclusive as done in line 108.